package/gluon-state/files/lib/gluon/state/check.d/has_default_gw6

```diff
@@ -0,0 +1,2 @@
+#!/bin/ash
+out=$(ip -6 route show default table all dev br-client 2>/dev/null) && [ -n "$out" ]
```

Comment:
Hmm, why the `table all`? The default table should be sufficient I think (and unless something has changed recently, the `table X` argument of busybox's `ip` implementation is extremely broken and shouldn't be used).

Reply:
It should be fixed if it is broken :smile:.

Reply:
Because @lemoer suggested it and I really favor explicit over implicit behaviour in programs.
It's less error prone and even while `ip` is not "extremely broken", using the explicit variant we would not have had to rely on it not changing its default output later.
Resolved in 270f1bb5e06377eabccc62aa6400dde3d537e9c4